src/controllers/mainController.ts

```diff
@@ -439,6 +442,19 @@ export default class MainController implements vscode.Disposable {
 					}
 				}
 			}));
+
+		const afService = new AzureFunctionProjectService(this.azureFunctionsService);
```

Comment:
Would suggest making this up with the rest of the services....

(but also as mentioned earlier, why is there not just a single AzureFunctionsService? Two seems unnecessary